action.yaml

```diff
@@ -54,15 +54,3 @@ inputs:
 runs:
   using: docker
   image: Dockerfile
-  args:
-    - ${{ inputs.command }}
-    - ${{ inputs.upstream_repo }}
-    - ${{ inputs.upstream_owner }}
```

Comment:
a little curious about why we can drop this?

Reply:
GitHub actions provides action inputs as environment variables `INPUT_<underscored name>`. The `entrypoint.sh` script is already using those environment variables, not positional arguments.